Content.Client/Chat/UI/SpeechBubble.cs

```diff
@@ -53,20 +53,20 @@ public static SpeechBubble CreateSpeechBubble(SpeechType type, string text, Enti
             switch (type)
             {
                 case SpeechType.Emote:
-                    return new EmoteSpeechBubble(text, senderEntity, eyeManager, chatManager, entityManager);
+                    return new TextSpeechBubble(text, senderEntity, eyeManager, chatManager, entityManager, "emoteBox");
 
                 case SpeechType.Say:
-                    return new SaySpeechBubble(text, senderEntity, eyeManager, chatManager, entityManager);
+                    return new TextSpeechBubble(text, senderEntity, eyeManager, chatManager, entityManager, "sayBox");
 
                 case SpeechType.Whisper:
-                    return new WhisperSpeechBubble(text, senderEntity, eyeManager, chatManager, entityManager);
+                    return new TextSpeechBubble(text, senderEntity, eyeManager, chatManager, entityManager, "whisperBox");
 
                 default:
                     throw new ArgumentOutOfRangeException();
             }
         }
 
-        public SpeechBubble(string text, EntityUid senderEntity, IEyeManager eyeManager, IChatManager chatManager, IEntityManager entityManager)
+        public SpeechBubble(string text, EntityUid senderEntity, IEyeManager eyeManager, IChatManager chatManager, IEntityManager entityManager, string speechClass)
```

Comment:
Can you rename it `speechStyleClass` to make it clearer what the variable is for?

Reply:
Absolutely.